README.md

```diff
@@ -106,18 +133,9 @@ Default value: `false`
 - **timeout:**  
 The amount of seconds after a task "end event" can be considered lost.  
 The task "map" is then evicted.  
-The default value is 0, which means no timeout so no auto eviction.
+The default value is 0, which means no timeout so no auto eviction.  
+
 
 ## Need Help?
 
 Need help? Try #logstash on freenode IRC or the https://discuss.elastic.co/c/logstash discussion forum.
```

Comment:
I like that, althought I think is also nice to have a short/small pointer to the build.md document, some sentence like, if you like to contribute, or build the plugin, see this.

What do you think?